src/components/Register/AccountDetails/SaveAccount/SaveAccount.js

```diff
@@ -84,7 +84,7 @@ export default class SaveAccount extends React.Component {
       return;
     }
 
-    const newAccount = new wallet.Account({ ...account, label });
+    const newAccount = new wallet.Account({ ...account, key: account.encryptedKey, label });
```

Comment:
I don't think we can change this and still have it considered a NEP6 wallet.  The [proposal](https://github.com/neo-project/proposals/blob/master/nep-6.mediawiki#account) states:

* `key` is the private key of the account in the NEP-2 format. This field can be null (for watch-only address or non-standard address).

An alternative to this would be to store encrypted keys to local storage once they are entered in the "Passphrase" tab, and allow those to appear in a dropdown in that tab after being used once.

Reply:
Hmm ok I get what you are saying, I'll go with your alternative method. Although i feel like there may be some errors with neon.js though in Account.js, it seems like the export function is taking the `_encrypted` variable and exporting it as the variable `key`, should it pehaps export it as `encryptedKey`? Is it because the AccountLike object does not have a `encryptedKey` field?
```
/**
   * Export Account as a AccountLike object.
   * @return {AccountLike}
   */
  export () {
    let key = null
    if (this._privateKey && !this._encrypted) throw new Error('Encrypt private key first!')
    if (this._encrypted) key = this._encrypted
    return {
      address: this.address,
      label: this.label,
      isDefault: this.isDefault,
      lock: this.lock,
      key,
      contract: this.contract,
      extra: this.extra
    }
  }
```

Reply:
Thanks for calling that out.  Perhaps I am unclear on the what the NEP6 proposal is saying.  I pinged snowypowers on Discord to ask about this since he wrote this code.  I'm interested in what he has to say.

If it's true that we can just use an encrypted version here, then this seems okay to me.  But it sounds like we would need to account for each account in the wallet being either a private key or an encrypted key.

Reply:
@kitchenSpoon Alright, I got some insight from snowy.

> NEP6 is always encrypted key or null (for watch only addresses)

So it seems we should have behavior for saving the wallet that's more in line with your original change here.  (I haven't gone through the additional commits yet.)

Reply:
@kitchenSpoon It looks like we can use `wallet.isNEP2` to see if it's encrypted.  Found [here](http://cityofzion.io/neon-js/docs/en/api-wallet.html#verify).

Reply:
@mhuggins thanks!, let me know how it goes with snowypowers when he replies you.